### DISCUSSION TITLE
If you were starting a new 2D web game...

### DISCUSSION THREAD

**Discussion by ghost** [enable3d/enable3d#345] [Polls]
Just a friendly poll. No favoritism meant or implied. You guys have used Enable3D and I just stumbled upon it which is strange because I follow Yandeu and am a supporter of his incredible libs (using geckos.io and typed-array-buffer-schema ATM..both are just awesome!).  So I was wondering if Enabled3D is good for a 2D only web game? (huge emphasis on 2D)

**Comment by yandeu** [comment]
If you only use 2d, I would recommend PixiJS.

**Reply by ghost** [reply]
Yeah, after some thought I was thinking this isn't a good poll. Tried to delete it but Github doesn't allow that. Which is very unintuitive in 2023. Very surprising coming from Github who is supposed to be at the bleeding edge of coding technology.
